README.md

```diff
@@ -0,0 +1 @@
+Recreated awesome books website, using ES6 syntax and modularizing JavaScript code
```

Comment:
- [ ] Kindly, Add a descriptive README file to your project,  A professional README file should have a project title, project description, list of technology used in the project, and instructions to set the project up locally, This includes `git clone URL` , `cd project_name`,  `npm install`, `npm start`, etc
Please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.
it will be great to add instructions on how to run the project. This includes `npm install` , `npm start` etc ...

Reply:
Well resolved 👍🏽